pkg/cloudprovider/cloudprovider.go

```diff
@@ -100,6 +104,37 @@ func New(ctx awscontext.Context) *CloudProvider {
 	}
 }
 
+func (c *CloudProvider) IsNodeDrifted(ctx context.Context, provisioner *v1alpha5.Provisioner, node v1.Node) (bool, error) {
```

Comment:
```suggestion
func (c *CloudProvider) IsNodeDrifted(ctx context.Context, provisioner *v1alpha5.Provisioner, node *v1.Node) (bool, error) {
```

Should be a pointer to the node so that we don't copy the node object around

Reply:
Done, thanks. Will push it.